app/models/notification.rb

```diff
@@ -55,15 +58,15 @@ class Notification < ActiveRecord::Base
   # ==========
 
   scope :active, (lambda do
-    where('starts_at <= :now and :now < expires_at', now: Time.now)
+    where('starts_at <= :now and :now < expires_at', now: Time.now).where(active: true)
   end)
 
   scope :active_per_user, (lambda do |user|
     if user.present?
       acknowledgement_ids = user.notifications.map(&:id)
-      active.where.not(id: acknowledgement_ids)
+      active.where.not(id: acknowledgement_ids).where(active: true)
```

Comment:
The `where(active: true)` should be unnecessary as this chains off the `active` method above